taxi_service/urls.py

```diff
@@ -14,8 +14,9 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path("", include("taxi.urls", namespace="taxi")),
 ]
```

Comment:
You didn't follow the instruction in the `Serving static files during development` section. Your code will not work in production
https://docs.djangoproject.com/en/4.1/howto/static-files/